### ISSUE TITLE
Ads are more annoying than they need to be

### ISSUE THREAD

**Issue by lemonnuggets** [Miodec/monkeytype#928]
**Describe the bug**
When you have ads turned on or on sellout mode, as the ads pop up and switch, the whole typing window moves around as well. The moving window makes it a lot harder to focus on typing and might discourage people from keeping the ads on.

**To Reproduce**
1.  Change ads setting to ON or SELLOUT
2. Use site normally

**Expected behavior**
I the active window was bounded to some fixed dimensions for each ad mode regardless of whether an ad is available or not, then this might not happen

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Brave
 - Version: 1.19.90

**Additional context**
![image](https://user-images.githubusercontent.com/63046776/107052621-e6b0ea80-67f3-11eb-8fbc-a789f17ebf81.png)

![image](https://user-images.githubusercontent.com/63046776/107052657-ee708f00-67f3-11eb-8a60-c5c1b76aae18.png)

Everything has to be pushed around and scaled for the ad at the bottom to show up which can be very distracting.

**Comment by TheeKevin** [comment]
Can't you just turn off ads if your annoyed by them? I get that you want to support Miodec while using the site but (at least for me) it hasn't really been a problem with the scaling.

**Comment by lemonnuggets** [comment]
@TheeKevin Yeah obviously I could just turn off the ads, but it looked like something that wasn't working the way it should have been so I created the issue. But yeah if it can't be reproduced by someone else then there isn't an issue here.
